atlas/commands/bootstrap/bootstrap.go

```diff
@@ -194,3 +203,12 @@ func initRepo() error {
 	fmt.Println("done!")
 	return nil
 }
+
+func initHelm(app application.Application) error {
+	fmt.Print("Generating helm files... ")
+	if err := runCommand("helm", "create", fmt.Sprintf("./helm/%s", app.Name)); err != nil {
+		return err
+	}
+	fmt.Println("done!")
```

Comment:
Does the default chart created by helm create match the default docker image, port,etc?

Reply:
This is still "work in progress". Added appropriate label and tagged PR